### PR TITLE
docs: add introduction, get started, and contributing pages to storybook

### PR DESCRIPTION
**Pull request by nathanyoung**
- Adds some basic pages for 
  - Introduction
  - Get Started
  - Contributing
- Sets the order of the items in the storybook left nav

Screen Shot 2020-10-26 at 3 05 03 PM

### CODE REVIEW

File: src/docs/GetStarted.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Get Started
+
+### 1. Install
+
+`yarn add @palmetto/palmetto-components`
+
+or
+
+`npm add @palmetto/palmetto-components`
```
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:13]: Should be `npm install @palmetto/palmetto-components`.

File: src/docs/Intro.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="About/Introduction" />
+
+# Palmetto Design System
+
+Welcome to the Palmetto Design System, whose purpose is to help us deliver high quality Palmetto branded applications, faster and better. It consists of the following following:
```
Comment (JCarmine) [inline src/docs/Intro.stories.mdx:7]: "following" is duplicated.

File: src/docs/GetStarted.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### 2. Import Global CSS
+
+```
```
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:17]: If we format this as jsx, it colorizes the line and makes it pop a little more.

File: src/docs/GetStarted.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+@import '@palmetto/palmetto-components/dist/css/utilities.css';
+@import '@palmetto/palmetto-components/dist/css/variables.css';
+```
+
+Not required but we also recommend importing our global reset in order to properly maintain the
+look of all components.
+
+```
```
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:25]: If we format this as js, it colorizes the line and makes it pop a little more. js and jsx have the same styling applied. So, either ```js or ```jsx.

File: src/docs/GetStarted.stories.mdx (hunk 3 above)
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:17]: If we format this as js, it colorizes the line and makes it pop a little more. ```js and ```jsx have the same styling applied. So, either ```js or ```jsx.

File: src/docs/GetStarted.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+@import '@palmetto/palmetto-components/dist/css/reset.scss'
```
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:26]: 1) Missing semi
2) Extension should `css`

File: src/docs/GetStarted.stories.mdx (hunk 1 above)
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:13]: Should be `npm install @palmetto/palmetto-components`

File: src/docs/GetStarted.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+`npm add @palmetto/palmetto-components`
+
+### 2. Import Global CSS
+
+```
+@import '@palmetto/palmetto-components/dist/css/utilities.css';
+@import '@palmetto/palmetto-components/dist/css/variables.css';
+```
+
+Not required but we also recommend importing our global reset in order to properly maintain the
```
Comment (JCarmine) [inline src/docs/GetStarted.stories.mdx:22]: Add the word "consistent" in here somewhere?

File: src/docs/Intro.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- [Palmetto Components](https://github.com/palmetto/palmetto-components) - on open source, react-based, component library for Palmetto applications and websites.
```
Comment (JCarmine) [inline src/docs/Intro.stories.mdx:9]: "an open source"

File: src/docs/Intro.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+- [Palmetto Design Tokens](https://github.com/palmetto/palmetto-design-tokens) - A central location to store shared attributes of the Palmetto Design System. These include things like color, fonts, spacing, and more.
+- Brand Assets: _coming soon_
+- Voice and Tone: _coming soon_
+
+If you have any questions or concerns, please feel to reach out to the UX Team on Slack (#ux). One of our missions is build tools to make your job easier and more succesful, so please don't hesitate to reach out.
```
Comment (JCarmine) [inline src/docs/Intro.stories.mdx:14]: "please feel free to"

Comment (JCarmine) [reply src/docs/Intro.stories.mdx:14]: Wording suggestion: "make your job easier and you more successful"?

Comment (JCarmine) [reply src/docs/Intro.stories.mdx:14]: "succesful" is misspelled. The following VSCode plugin is fantastic at catching these things: https://github.com/streetsidesoftware/vscode-spell-checker.

Er, actually, it doesn't work for `.mdx` files, but works great for most other formats.

Comment (JCarmine) [reply src/docs/Intro.stories.mdx:14]: On a more foundational level. Who is the audience for this repo? In the README, we are speaking to developers around the world, hoping that they will use it, and contribute, which is why it's open source. 

However, this line assumes that the audience is made-up of internal Palmetto employees.

Not a pressing matter, but more something to be pondered going forward.

**Review by JCarmine — CHANGES_REQUESTED** [review]

File: src/docs/Intro.stories.mdx (hunk 8 above)
Comment (nathanyoung) [reply src/docs/Intro.stories.mdx:14]: most definitely Palmetto employees. a very distant second would be for engineers we are trying to recruit.

**Review by juanfabrega — APPROVED** [review]

File: src/docs/GetStarted.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+look of all components across applications.
+
+```jsx
+@import '@palmetto/palmetto-components/dist/css/reset.css'
+```
+
+### 3. Usage
+
+```jsx
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (nathanyoung) [reply src/docs/GetStarted.stories.mdx:32]: added it back in

**Review by JCarmine — APPROVED** [review]

**Pull request merged**